include/rospack/rospack.h

```diff
@@ -152,6 +152,7 @@ class ROSPACK_DECL Rosstackage
     std::tr1::unordered_map<std::string, Stackage*> stackages_;
     Stackage* findWithRecrawl(const std::string& name);
     void log(const std::string& level, const std::string& msg, bool append_errno);
+    void clearStackages();
```

Comment:
We don't enforce it but try to avoid it when possible since many more people have from-source builds.